src/converter.ts

```diff
@@ -401,24 +448,44 @@ export class Converter {
       }
 
       const screenshot = async (pageNumber = 1) => {
-        const clip = {
-          x: 0,
-          y: (pageNumber - 1) * tpl.rendered.size.height,
-          ...tpl.rendered.size,
-        } as const
+        const y = (pageNumber - 1) * viewPort.height
+        const clip = { x: 0, y, width: viewPort.width, height: viewPort.height }
+
+        if (browser.protocol === 'cdp') {
+          if (opts.type === ConvertType.jpeg)
+            return await page.screenshot({
+              clip,
+              quality: opts.quality,
+              type: 'jpeg',
+            })
 
-        if (opts.type === ConvertType.jpeg)
           return await page.screenshot({
             clip,
-            quality: opts.quality,
-            type: 'jpeg',
+            omitBackground: true,
+            type: 'png',
           })
+        } else if (browser.protocol === 'webDriverBiDi') {
+          if (browser.kind === 'firefox') {
+            clip.y = 0
 
-        return await page.screenshot({
-          clip,
-          omitBackground: true,
-          type: 'png',
-        })
+            await page.evaluate(
+              `document.body.scrollTo({ left: 0, top: ${y}, behavior: 'instant' })`
+            )
+          }
+
+          // WebDriver BiDi only supports PNG
+          const buf = await page.screenshot({ clip, type: 'png' })
+
+          if (opts.type === ConvertType.jpeg) {
+            // Convert png to jpeg via sharp
+            return await sharp()(buf)
+              .jpeg({ mozjpeg: true, quality: opts.quality })
+              .toBuffer()
```

Comment:
Use `sharp` to convert PNG to JPG because WebDriver Bidi does only support PNG screenshot.

- **ToDo**
  - [ ] Check compatibility with the standalone binary